build-system/tasks/release/index.js

```diff
@@ -116,6 +116,12 @@ const CHANNEL_CONFIGS = {
   '25': {type: 'experimentC', configBase: 'prod'}, // Spec name: 'inabox-experimentC'
 };
 
+// Paths to custom configurations
+const CUSTOM_FLAVORS_CONFIG_PATH =
+  'build-system/global-configs/custom-flavors-config.json';
+const CUSTOM_OVERLAY_CONFIG_PATH =
+  'build-system/global-configs/custom-config.json';
```

Comment:
Let's do the path.resolve here instead of inside the functions:

```suggestion
const CUSTOM_FLAVORS_CONFIG_PATH = path.resolve(
  __dirname,
  '../../../build-system/global-configs/custom-flavors-config.json'
);
const CUSTOM_OVERLAY_CONFIG_PATH = path.resolve(
  __dirname,
  '../../../build-system/global-configs/custom-config.json'
);
```

Reply:
Unfortunately, `amp clean` does not support absolute paths.

Reply:
Oh I see, I missed that this gets used in two different places

Reply:
In commit 8bde827, `amp clean` exclusions are no longer modified in `build-system/tasks/release/index.js`, so this change has been applied.